conf/application.conf

```diff
@@ -119,6 +120,7 @@ braingames.binary {
 webKnossos {
   user.time.tracingPauseInSeconds = 60
   tasks.maxOpenPerUser = 5
+  tasks.reopenAllowedBackend = 300
```

Comment:
I think it would be better to just use a single config entry for this, rather than two. Is there a use-case where those numbers should differ? Feel free to access the config value in `features` from the backend also. In fact, maybe we should consider renaming it at some point…
speaking of renaming, I agree that this config value / variable should have `inSeconds` in the name to clarify both that it is a duration and to suggest the unit :)